torch/_dynamo/variables/functions.py

```diff
@@ -1169,10 +1169,11 @@ def call_function(
             torch._dynamo.utils.warn_once(msg)
             unimplemented(msg)
         else:
+            qualname = self.value.__qualname__
```

Comment:
Couldn't this line still crash if `__qualname__` doesn't exist?

Reply:
+1 I don't see how this fixes the error.

Reply:
Sorry was writing this in the middle of a SEV and screwed it up - Code is fixed (and a couple more callsites added).